dashboard/README.md

```diff
@@ -4,53 +4,49 @@
 
 Pbench Dashboard is a web-based platform for consuming indexed performance benchmark data. The platform provides a consolidated view of benchmark data within tables, charts, and other powerful visualizations. Users are able to quickly navigate through benchmark data and tune analytics through comparison tools for in-depth analysis.
 
-  
+# Dashboard directory structure
 
-## Scaffolding
+## public
```

Comment:
Section heads which refer to specific directories would probably be better in mono-spaced text.  And, it might be good to "decorate" them, e.g.:
```
## The `public` directory
```

Reply:
Updated